goecheck.sh

```diff
@@ -17,10 +17,16 @@ goecheck(){
 					curl --silent --connect-timeout $goetimeoutlp1 -s http://$goeiplp1/mqtt?payload=alw=0 > /dev/null
 				fi
 			fi
+			fwv=$(echo $output | jq -r '.fwv')
+			printf -v fwv %0.f "$fwv"
```

Comment:
Wurde das in einer aktuellen Installation geprüft? Ich erhalte immer die Fehlermeldung `printf: 040.0: Ungültige Oktalzahl`.

Reply:
Ja, kann den output leider nicht eliminieren, das Ergebniss stimmt aber.

Reply:
Alternativ fwv=$(echo $fmw | grep -Po "[1-9]\d{1,2}")

Reply:
Vielleicht ist so tatsächlich die bessere Variante

fwv=$(echo $output | jq -r '.fwv' | grep -Po "[1-9]\d{1,2}")